clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py

```diff
@@ -101,7 +103,9 @@ def create_in_mem_container(
         return container
 
     def execute(self):
-        timestamp = self.context.start.strftime("%Y%m%dT%H%M%S")
+        start = self.context.start
```

Comment:
What if a lims-user decides to open up the create samples step, and then never exit it? (She doesn't have to exit it between subsequent logins) Is the "start" timestamp going to be the same all the time?

Reply:
No. This is the start of the execution of the script.